components/5cards.html

```diff
@@ -0,0 +1,174 @@
+                    </div>
+                </div>
+                <div class="carbon div-ch">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=paraiso-dark&wt=none&l=javascript&ds=true&dsyoff=20px&dsblur=63px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2509%253Cdiv%2520class%253D%2522card%2522%253E%250A%2509%2509%253Cspan%2520class%253D%2522prod-badge%2522%253ETrending%253C%252Fspan%253E%250A%2509%2509%253Cimg%2520class%253D%2522prod-img%2522%2520src%253D%2522%252Fimages%252Fm-cc201-dustypink-dennis-lingo-original-imafueutmgyvtfdr.jpeg%2522%2520alt%253D%2522%2522%253E%250A%2509%2509%253Ch3%253EProduct%2520name%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522price%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2509%253Cp%253E%25E2%2582%25B9%2520999%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522price-orig%2522%253E%25E2%2582%25B9%25201599%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253E%2845%2525%2520off%29%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253ERatings%253C%252Fp%253E%250A%2509%2509%253Cdiv%2520class%253D%2522prod-btns%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2509%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EAdd%2520to%2520Cart%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2509%253Cbutton%2520class%253D%2522btn%2522%253E%25E2%259D%25A4%25EF%25B8%258F%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2509%253C%252Fdiv%253E"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <h2 class="div-ch">Horizontal card</h2>
+                <div class="code sub-cont div-ch">
+                    <div class="card-horiz">
+                        <span class="prod-badge">Trending</span>
+                        <img class="prod-img-horiz"
+                            src="/images/m-cc201-dustypink-dennis-lingo-original-imafueutmgyvtfdr.jpeg" alt="">
+                        <div class="card-sub-div-horiz">
+                            <h3>Product name</h3>
+                            <div class="price">
+                                <p>₹ 999</p>
+                                <p class="price-orig">₹ 1599</p>
+                                <p>(45% off)</p>
+                            </div>
+                            <p>Ratings</p>
+                            <div class="prod-btns">
+                                <button class="btn btn-primary">Add to Cart</button>
+                                <button class="btn">❤️</button>
+                            </div>
+                        </div>
```

Comment:
On frontend, this content has a lot of blank space so either increase info or decrease the card height

Reply:
Done